roles/contiv_network/tasks/main.yml

```diff
@@ -68,3 +68,6 @@
   args:
     chdir: /usr/bin/
     creates: contivctl
+
+- include: aci_tasks.yml
+  when: (run_as == "master") and (aci_mode is defined) and (aci_mode|bool)
```

Comment:
Assuming `false` is a reasonable default for `aci_mode`, we shall define it in `group-vars/all` and default it to `false` there.

Minor but since `contiv_network` is going to have multiple deployment modes viz aci, standalone-with-vlan, standalone-with-bgp and so on. It might be better to have a string variable like `contiv-network-mode` and set it to a resonable default in `group-vars/all` along with all possible values in a comment there.